docs/developer/extending/apps/developing-apps/app-sdk/api-handlers.mdx

```diff
@@ -1,8 +1,12 @@
 # API Handlers
 
-Saleor Apps are meant to work in a serverless environment, where Cloud Functions are the foundations of server-side code.
+App SDK mainly targets serverless environments, like Vercel or AWS Lambda. It provides a set of handlers that can be used to build Saleor apps.
 
-Currently, Saleor heavily relies on Next.js, but other platforms will be supported in the future.
+SDK providers helpers for following types of functions (that represent API endpoints):
+- Manifest handler - Used to fetch app manifest by Saleor during app installation.
```

Comment:
suggestion: we could link to this page with more details:

https://docs.saleor.io/developer/extending/apps/architecture/app-requirements#manifest-url